Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -131,6 +141,10 @@ public function apply(ResourceInterface $resource, QueryBuilder $queryBuilder, R
                 }
 
                 $strategy = null !== $this->properties ? $this->properties[$property] : self::STRATEGY_EXACT;
+                if (strpos($strategy, 'i') === 0) {
+                    $strategy = substr($strategy, 1);
+                    $this->caseSensitive = false;
```

Comment:
Because you are using a class property, that means that if we use the same class for 2 requests (which can definitely be the case, either in tests, reactphp or with sub requests) then the ones after the first case insensitive request will also be case insensitive...

I wouldn't use a property but simply a local variable that you pass to the `caseWrap` method.